package.json

```diff
@@ -109,6 +109,7 @@
     "electron-timber": "^0.5.1",
     "electron-updater": "^4.2.2",
     "flat": "^5.0.0",
+    "flow-geojson": "^2.0.7",
```

Comment:
is this still needed? (and `@babel/preset-flow`?) @gmaclennan

Reply:
These are both needed for the type checking (we use types from flow-geojson and babel preset allows linting and compile to work on code with flow.)